packages/neocmakelsp/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/Decodetalkers/neocmakelsp
+TERMUX_PKG_DESCRIPTION="the lsp for cmake"
```

Comment:
Please use one of the descriptions already in use for this package.
https://repology.org/project/neocmakelsp/information